### ISSUE TITLE
Two way communication

### ISSUE THREAD

**Issue by fif911** [PowerBroker2/SerialTransfer#106]
I am new to this library. But I have not found a way to achieve two way I2C communication between Master and Slave. Is it possible with the current implementation?

**Comment by PowerBroker2** [comment]
Yes, there are I2C examples in the [examples folder](https://github.com/PowerBroker2/SerialTransfer/tree/master/examples)

**Comment by fif911** [comment]
I looked through them, but I cannot find a 2-way communication example. I was able to find only Master to Slave but not Slave to Master (can be done with requestFrom in wire lib). Is there a way to do smth like this? @PowerBroker2

**Comment by Drikteur** [comment]
Also looking for 2 way communication on uart, tried to merge the sketches (TX and RX)and it works both ways but not at the same time, the arduino which boots latest acts as master... (trying all day but to stupid to make it work :-))
